### PR TITLE
feat: Integration Training Event Filter Table

### PR DESCRIPTION
**Pull request by jeheskielSunloy77**
Search Filters demo
https://user-images.githubusercontent.com/38202065/188445282-fc846b84-3f93-498d-af3e-4d4cbc11ce66.mp4

### CODE REVIEW

File: src/pages/Home/sections/AllTrainings/index.jsx [hunk 1]
```diff
@@ -10,10 +10,15 @@ import axios from 'axios'
 import { useContext, useEffect, useState } from 'react'
 
 const AllTrainings = () => {
-	const { dataView } = useContext(AppContext)
+	const { dataView, searchParams } = useContext(AppContext)
 	const [allTrainingData, setAllTrainingData] = useState([])
 	const [pageLimit, setPageLimit] = useState(5)
 	const [dataLength, setDataLength] = useState(0)
+	const url = `trainings?${
```
Comment (argorfc) [inline src/pages/Home/sections/AllTrainings/index.jsx:17]: move the constant into utils function to serialize query params dynamically

File: src/pages/Home/sections/MyTraining/index.jsx [hunk 2]
```diff
@@ -8,13 +8,21 @@ import { lazy, Suspense, useContext, useEffect, useState } from 'react'
...
 
 const MyTraining = () => {
-	const { dataView } = useContext(AppContext)
+	const { dataView, searchParams } = useContext(AppContext)
 	const [myTrainingData, setMyTrainingData] = useState(null)
+	const token = localStorage.getItem('token')
+	const { userId } = jwt_decode(token)
+	const date = moment(new Date()).format()
+
+	const url = `trainings?${
```
Comment (argorfc) [inline src/pages/Home/sections/MyTraining/index.jsx:17]: move the constant into utils function to serialize query params dynamically

**Review by argorfc — CHANGES_REQUESTED** [review]
Challenges today, remove duplicate code.

**Comment by jeheskielSunloy77** [comment]
@argorfc i have update the pull request

**Review by argorfc — APPROVED** [review]
Approved.

**Review by argorfc — CHANGES_REQUESTED** [review]
Need update the query params

**Comment by jeheskielSunloy77** [comment]
@argorfc i have update the pull request and have change the queryParams function

**Review by argorfc — APPROVED** [review]
Approved

**Pull request merged**